document/static/js/modules/document_template/fix_doc.js

```diff
@@ -151,7 +151,7 @@ export function adjustDocToTemplate(doc, template, documentStyles, citationStyle
                 )
             ) {
                 if (newNode.attrs.initial) {
-                    newNode.content = newNode.attrs.initial
```

Comment:
@anandksalien This does not look right - we should make the document have the content specified in the template's "initial" field, but you are instead making the template take over the node's content.

Reply:
@anandksalien Could you explain what you are trying to achieve here and in what cases Ids in TOCs are not created correctly?